### PR TITLE
Ray operator build

### PR DESCRIPTION
**Pull request by erikerlandson**
## Related Issues and Dependencies

https://github.com/thoth-station/core/issues/311

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements
This is the first build of a ray operator image for thoth-station

## Description
Add and configure all the files needed to do a `Containerfile` build strategy.

### CODE REVIEW

**Comment by erikerlandson** [comment]
@harshad16  the `build-check` is completing with:
`Tasks Completed: 4 (Failed: 0, Cancelled 0), Skipped: 6` 

And says it is in "pending" state - how should I interpret this?

**Comment by harshad16** [comment]
we have to have the first pr merged, to run the build-check, as it looks for some component in the repo.
like the check: `thoth-build` , if that in the repo, then it run pr build-check properly.

**Comment by erikerlandson** [comment]
@harshad16  so should we merge this, and make any corrections with new PRs?

cc @goern

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-#   build-source-script: "image:///opt/app-root/builder"
-#   base-image: quay.io/thoth-station/s2i-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+check:
+  - thoth-build
+build:
+  build-strategy: Containerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
```
Comment (harshad16) [inline .aicoe-ci.yaml]: lets remove the comments.

**Comment by erikerlandson** [comment]
with that in mind, would it be preferable to push a degenerate or 'empty' PR, where the Containerfile just has a FROM, and the Pipfile is empty, etc?

**Review by harshad16 — COMMENTED** [review]
lgtm
a small comment

File: .aicoe-ci.yaml (hunk 1 above)
Comment (erikerlandson) [reply .aicoe-ci.yaml]: :+1:

Comment (erikerlandson) [reply .aicoe-ci.yaml:6]: I would actually be interested in having all these yaml files have LOTS of comments, describing each field, or at least one comment pointing future contributors to full documentation of all fields.

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**